tmk_core/protocol/chibios/usb_main.c

```diff
@@ -610,6 +616,14 @@ static void set_led_transfer_cb(USBDriver *usbp) {
     }
 }
 
+#ifdef MOUSE_SCROLL_HIRES_ENABLE
+static void set_multiplier_cb(USBDriver *usbp) {
+    if (usbp->setup[6] == 2 && set_report_buf[0] == REPORT_ID_MULTIPLIER) {
+        resolution_multiplier = set_report_buf[1]
```

Comment:
I think this is missing a ";"

Reply:
Yup,
Thanks for testing this!
Will add in the next update I did not even test if the vusb or ChibiOS code compiled...